### ISSUE TITLE
Binary Output

### ISSUE THREAD

**Issue by mbhurtel** [llSourcell/A_Guide_to_Running_Tensorflow_Models_on_Android#7]
**export_model(tf.train.Saver(), model, ["conv2d_1_input"], "dense_2/Softmax")**

In the above line of code, what would be the parameters for Binary output?
I have used only ONE convolution layer.

I just want to make everything clear before training.
Thanking you in anticipation.
